packages/bridge/src/reactBridge.ts

```diff
@@ -18,12 +18,46 @@ export let GarfishContext = null;
 //   // ignore
 // }
 
+type RequiredOpts = {
+  React: any;
+  ReactDOM: any;
+};
+
+type OptionalOpts = {
+  renderType: Function | string;
+  errorBoundary: (e) => any;
```

Comment:
also errorBoundary should be `(caughtError: boolean, info:SomeInfo, props: any)=>React.Element|null`

<img width="685" alt="image" src="https://user-images.githubusercontent.com/9161085/156171726-bf642e34-181f-4781-941a-f211777ac0c9.png">